apps/app/src/utils/pageScroll.ts

```diff
@@ -1,13 +1,23 @@
 import "client-only";
 
-const classes = ["overflow-hidden"];
-
-export const lockScroll = () => {
+export const lockScroll = (mobileOnly = false) => {
```

Comment:
Ogólnie, parametry typu `boolean` są kiepskie bo przy wywołaniu nie jest od razu klarowne co taki parametr robi. Lepiej użyć obiektu:
```suggestion
export const lockScroll = ({ mobileOnly }: { mobileOnly: boolean }) => {
```